h/static/styles/variables.scss

```diff
@@ -35,7 +35,7 @@ $link-color-hover:       $hypothered !default;
 
 // Typography
 // -------------------------
-$sans-font-family:        "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif !default;
+$sans-font-family:        "Lato","Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif !default;
```

Comment:
Yep. Sorry, I was going to to do that separately, but forgot to remove that instance.